Networking/Networking/Remote/FeatureFlagRemote.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+
+/// Protocol for `FeatureFlagsRemote` mainly used for mocking.
+///
+public protocol FeatureFlagRemoteProtocol {
+    func loadAllFeatureFlags() async throws -> [RemoteFeatureFlag: Bool]
+}
+
+/// Feature Flags: Remote Endpoints
+///
+public class FeatureFlagRemote: Remote, FeatureFlagRemoteProtocol {
+    public func loadAllFeatureFlags() async throws -> [RemoteFeatureFlag: Bool] {
+        let parameters: [String: String] = [
+            ParameterKeys.platform: "ios",
```

Comment:
Super nit: 
I see that we use `iOS` [here](https://github.com/woocommerce/woocommerce-ios/blob/trunk/WooCommerce/Classes/Analytics/TracksProvider.swift#L101) in `TracksProvider`. If possible, it will be nice to follow the same capitalization pattern here as well.

Reply:
this is true, but other apps already use this parameter in production, and I don't think we can easily change that.

(I can't share backend code links here, you can look at the existing feature flags from JPiOS/WPiOS in the file `mobile-apps.php` in the WPCOM codebase)